charter/00-CHARTER.md

```diff
@@ -0,0 +1,227 @@
+        At least three (3) and up to five (5) members may be appointed to share the decision
+        making authority of Department operations and its vote on the Council;
+        each Delegate is assigned control over a portion of the Department vote on the 
+        Council;
+    b) Representative: 
+        appointed by the Delegates, this position is responsible for attending and 
+        representing the Department in any Council meetings or matters;
+        Departments may assign up to TWO (2) alternate Representatives that will be
+        recognized in order to engage in official business.
+    c) Contributors: 
+        members assigned day-to-day operational tasks or project work;
+        Contributors may vote on internal Department affairs as determined by the 
+        Department’s Operating Policy;
+
+### Section V.6 --- Council Representation
+
+Each Department must assign a Representative to engage in the affairs of the Council on behalf of the Department to ensure comprehensive participation and effective communication during the session.
+
+Departments may also choose to declare up to two (2) alternate Representatives to engage in Council affairs on behalf of the Department. Alternates must be assigned their role using the same process as the Representative role and identified as "Alternate Representative" and, in the case of two (2) Alternates, "First Alternate Representative" and "Second Alternate Representative" to clarify the rank of the Representative. 
+
+Duties of Representation are first assigned to the Representative, if they are not available or unable to fulfill the responsibilities then the First Alternate may represent the Department.  If the Representative and First Alternate are not available then the Second Alternate may represent the Department in Council business.
+
+### Section V.7 --- Minimum Viable Membership
+To be considered an active department, membership must include a minimum of three (3) members who have been elected to the Delegate role.
+
+### Section V.8 -- Active Department Status
+Departments failing to meet Minimum Viable Membership requirements [as defined by Section V.7] shall not be accorded active status, thereby rendering them ineligible to exercise voting privileges on Council matters. Nonetheless, they shall persist in contributing to the calculation of Council quorum.
+
+### Section V.9 — Conflicts of Interest
+
+Each Delegate must, upon their appointment, publicly submit a disclosure of any conflicts of interest to the entirety of the Council membership.
+
+Delegates must maintain and keep current disclosures in the event of the emergence of new conflicts. In cases where a Delegate acknowledges a conflict of interest pertaining to a particular undertaking or subject matter, the delegate must recuse themselves from any deliberations or discussions concerning the matter. Additionally, the delegate must cast an 'Abstain' vote on any matters directly related to the identified conflict.
```

Comment:
I would remove the `or discussions`, since their contribution can still be valuable (as happened with Dimi on the community DEX already). Eg. the new COI policy draft proposal says `the member might participate in public discussions regarding such proposals only after remarking his COI.`